### DISCUSSION TITLE
[#958959/1062848] Selection corners or "resize gizmo's" are not visible through blocks

### DISCUSSION THREAD

**Discussion by Landy005** [Mojang/minecraft-editor#51] [Bugs]
The selection box currently has it's pivotable corners or "Gizmo's" set as a white little box. But that box is not visible when it's behind or in other blocks. Having these showing through would help clicking on them to further edit the selection. 

Was this done on purpose to not show, if so for what reason?

Steps to recreate;

1 Make a selection
![image](https://user-images.githubusercontent.com/16018643/223969002-3ca560ad-743f-432c-b5d5-c28e1807a14d.png)
2 Extend the corner/gizmo through other blocks or terrain
3 Observe the white boxes and pivot dissapearing behind the blocks
![image](https://user-images.githubusercontent.com/16018643/223969114-ad0083bb-a663-4d81-8ba1-8e7fb54b89d1.png)

Expected result; 
Pivot and white boxes/gizmos showing through the blocks regardless.
I feel like the whole selection box and it's color represent the depth and 3D aspect of the selection enough with it's interraction with blocks, but the corners should stay visible.

**Comment by CaptainCooky** [comment]
This is also a major issue for the move selection gizmo. It is unable to be selected if it's obscured by blocks.

**Comment by minecraftchloe** [comment]
+1! We have work items on this -- partnership with Design and our amazing Graphics team 💪 Updating the issue with IDs for better tracking. 

Fantastic documentation, by the way!

**Comment by dacowan** [comment]
We’re aware of this one; in the mean time, you can use ALT/CTRL Arrow Keys and PGUP/DN to manually move the selection volume around, and resize it in each dimension — for those times you can’t see the gizmo

**Comment by minecraftchloe** [comment]
We fixed this!
